openqaoa/workflows/optimizer.py

```diff
@@ -19,12 +19,12 @@
 from typing import List
 import gzip
 
+from openqaoa.problems import QUBO
 from openqaoa.devices import DeviceLocal, DeviceBase
-from openqaoa.problems.problem import QUBO
 from openqaoa.workflows.parameters.qaoa_parameters import CircuitProperties, BackendProperties, ClassicalOptimizer
 from openqaoa.workflows.parameters.rqaoa_parameters import RqaoaParameters
 from openqaoa.qaoa_parameters import Hamiltonian, QAOACircuitParams, create_qaoa_variational_params
-from openqaoa.utilities import get_mixer_hamiltonian, ground_state_hamiltonian, exp_val_hamiltonian_termwise, delete_keys_from_dict, is_valid_uuid, generate_uuid
+from openqaoa.utilities import get_mixer_hamiltonian, ground_state_hamiltonian, exp_val_hamiltonian_termwise, delete_keys_from_dict, is_valid_uuid, generate_uuid, convert2serialize
```

Comment:
Do we need to import `convert2serialize` ?

Reply:
Removed in latest commit. Its not needed.